tools/proofers/image_block_enh.inc

```diff
@@ -47,143 +29,38 @@ function ibe_get_styles()
         $textLeft = "1%";
     }
 
-    $minor_width = '25px';
-    $minor_height = '25px';
-
-    $major_width = ($block_width_pc - 3) . '%';
-    $major_height = ($block_height_pc - 3) . '%';
-
     // (This assumes that 25px is about 3% of either dimension of the proofframe;
     // i.e., that the proofframe is roughly 800 x 800px.)
     return <<<STYLES
-            #controlframe {
-                position:absolute;
-                left:$textLeft;
-                top:$textTop;
-                width:$textWidth%;
-                height:$textHeight%;
-                /* clip:rect(0px, 100%, 100%, 0px); -- broken in MS Edge */
-                text-align:center;
-                z-index:6;
-                overflow:auto;
-            }
-
-            #imagehorz {
-                position:absolute;
-                left:$minor_width;
-                top:0px;
-                width:$major_width;
-                height:$minor_height;
-                z-index:3;
-            }
-            #imagevert {
-                position:absolute;
-                left:0px;
-                top:$minor_height;
-                width:$minor_width;
-                height:$major_height;
-                z-index:4;
-            }
-            #imageframe {
-                position:absolute;
-                top:$minor_height;
-                left:$minor_width;
-                width:$major_width;
-                height:$major_height;
-                /* clip:rect(0px, 100%, 100%, 0px); -- broken in MS Edge */
-                z-index:2;
-                text-align:center;
-                overflow:auto;
-            }
-            #imagedisplay {
-                position:absolute;
-                left:0px;
-                top:0px;
-                z-index:1;
-            }
+                #controlframe {
```

Comment:
This is very picky, and I don't know that I have a strong opinion one way or the other, but is using "frame" in the id (here and for `#newimageframe`) appropriate since we're moving away from coding using frames?

Reply:
You are right. These aren't frames but divs.